CONTRIBUTING.md

```diff
@@ -19,6 +19,28 @@ This guide helps make contributing to Syna Theme simple and efficient for everyo
 If you are starting out contributing to Syna Theme, there might be smaller scoped "starter issues" available:
 [Current starter issues](https://github.com/okkur/syna/labels/starter%20issue)
 
+Any changes in CSS/JS files or fonts require webpack which is managed with `package.json`'s npm scripts. You need to run the following commands which would install the required dependencies for building the changed files and start webpack for the build process itself.
+
+```
+yarn install
+yarn start
```

Comment:
We have that within the makefile so we should probably use the make commands here or?